homeassistant/helpers/icon.py

```diff
@@ -29,3 +29,23 @@ def icon_for_signal_level(signal_level: int | None = None) -> str:
     if signal_level > 30:
         return "mdi:signal-cellular-2"
     return "mdi:signal-cellular-1"
+
+
+def icon_for_volume_level(
```

Comment:
Where will we use this, for what entity type? Is it number entities? Maybe we should add a device class instead and let the frontend take care of the icon?

Reply:
Media player (based on the screenshots and  a discussion  @ frontend repo)

Reply:
That is probably better, however, number entities don't support device classes at this point.

Reply:
I'd support adding that if we have multiple integrations implementing the same kind of number entities.

Reply:
I updated the PR description to explain better.

Reply:
We have discussed this internally, and are actually in favor of handling these things via the device classes, as this is a frontend concern (not backend).

Reply:
Okay, I'll close this PR.
Thanks.